worlds/house.world

```diff
@@ -11,7 +11,7 @@
     </include>
 
     <physics type="ode">
-      <real_time_update_rate>1000.0</real_time_update_rate>
+      <real_time_update_rate>0000.0</real_time_update_rate>
```

Comment:
Is this supposed to be here?

Reply:
That one is just my personal preference, got added by mistake